code/lib/core-server/src/utils/StoryIndexGenerator.test.ts

```diff
@@ -50,12 +52,24 @@ const csfIndexer = async (fileName: string, opts: any) => {
   return loadCsf(code, { ...opts, fileName }).parse();
 };
 
+// const storiesMdxIndexer = async (fileName: string, opts: any) => {
+//   const code = (await fs.readFile(fileName, 'utf-8')).toString();
+//   const { compile } = await import('@storybook/mdx2-csf');
+//   try {
+//     console.log(code, compile);
+
+//     code = await compile(code, {});
+//   } catch (err) {
+//     console.log(err);
+//     throw err;
+//   }
+//   return loadCsf(code, { ...opts, fileName }).parse();
+// };
+
 const options = {
   configDir: path.join(__dirname, '__mockdata__'),
   workingDir: path.join(__dirname, '__mockdata__'),
-  storyIndexers: [
-    { test: /\.stories\..*$/, indexer: csfIndexer as any as CoreCommon_StoryIndexer['indexer'] },
-  ],
+  storyIndexers: storyIndexers([{ test: /\.stories\.(js|ts)x?$/, indexer: csfIndexer }]),
```

Comment:
@shilman this and the commented out snippet above both error in a similar inscrutable way in the changed test below. Can you take a look?